src/containers/Rewards/index.tsx

```
@@ -0,0 +1,1166 @@
+  };
+
+  const getBadgeColors = (key: string) => {
+    switch (key) {
+      case 'bonus':
+        return { bg: 'rgba(247,198,69,0.15)', txt: '#F7C645' };
+      case 'spends':
+        return { bg: 'rgba(38,133,202,0.15)', txt: '#2685CA' };
+      case 'merchant':
+        return { bg: 'rgba(226,92,92,0.15)', txt: '#E25C5C' };
+      case 'referrals':
+      default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)